docs/FAQ.rst

```diff
@@ -316,6 +332,25 @@ If using Docker, specify the URI as the name of the Docker container running
 the Avalon Listener:  ``http://avalon-listener:1947`` on the
 command line (the option is usually ``--uri`` or ``--service-uri``).
 
+How do I fix ``RuntimeError`` when I make changes to trusted code in enclave managers (KME or Singleton) and try to run?
```

Comment:
Another option is to bind sealed data to MRSIGNER instead of MRENCLAVE. This will allow to use the old sealed data even if MRENCLAVE value of KME changea.